### ISSUE TITLE
Should not all lab times be at 4pm? 

### ISSUE THREAD

**Issue by TimLanglois** [UWA-SCIE2204-Marine-Systems/How-to-complete-labs#1]
@Matt-Navarro 
Should not all lab times be at 4pm? to avoid overload of ecocloud?

**Comment by Matt-Navarro** [comment]
I don't think we can change the lab times? Students have scheduled their availability around the original lab times.


________________________________
